srcpkgs/protonvpn-cli/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'protonvpn-cli'
 pkgname=protonvpn-cli
 version=2.2.2
-revision=1
+revision=2
```

Comment:
a rebuild is not necessary i guess, so please don't revbump the package

Reply:
I'll fix this first thing in the morning. I was unsure since the checksum changed.

Reply:
Should be good now. I'll keep that in mind for upstream changes in the future.